.github/workflows/any-pr.yml

```diff
@@ -1,9 +1,10 @@
-name: CI
+name: any-pr
```

Comment:
The name should describe what the action does, not when it is triggered IMO.
For example: "Continous Integration" or "Run tests"

Reply:
I noticed this as well... I can see some advantage to a a single workflow per "when it is triggered", but seems like https://help.github.com/en/actions/reference/events-that-trigger-workflows

its probably a best practice to have the workflow functionality named, and doing one thing, and then have multiple workflows run `on: [push, pull_request]`, etc...

Reply:
Happy to rename however I sort of disagree, because in this workflow setup where there are not multiple actions per GH workflow it tends to provide the right level of detail to the developer in the name, rather than a general term like CI that most developers have a different definition of.

How about this as a proposal

any-pr => ci
push-master => cd - importantly the d here means delivery not deployment
push-release => release

Reply:
these seem more in line with what I would expect, but as you note... if there are only ever going to be these workflows the file name is not that important... what is important is what shows up on the PR Preview, when things get more complicated.... for example:

<img width="493" alt="Screen Shot 2020-06-27 at 2 52 22 PM" src="https://user-images.githubusercontent.com/8295856/85930924-ddd36000-b885-11ea-872a-000e5aadd701.png">

... I hope most developers think that CI = Lint + Tests + Build, but I guess that assumption might not be safe.

Reply:
Its not a hill I would die on ever, just a comment :)

Reply:
Have updated now, let me know what you guys think @gjgd @OR13